### PR TITLE
Fix crash bug due to image decompression memory consumption

### PR DESCRIPTION
**Pull request by coreyfloyd**
Also remove anything that was dowloaded in the background from the memory cache

### CODE REVIEW

**Comment by coreyfloyd** [comment]
https://phabricator.wikimedia.org/T142602

**Comment by coreyfloyd** [comment]
accidentally merged this to dev already… 😞

File: Wikipedia/Code/WMFImageController.swift [hunk 1]
```diff
@@ -454,6 +454,7 @@ extension WMFImageController {
             }
 
             let success = { (didCache: Bool) in
+                self.imageManager.wmf_removeImageForURL(imageURL, fromDisk: false)
```
Comment (montehurd) [inline Wikipedia/Code/WMFImageController.swift]: Does this mean if we've successfully cached an image, we're then going to remove it from the cache?

**Comment by joewalsh** [comment]
👍

File: Wikipedia/Code/WMFImageController.swift (hunk 1 above)
Comment (montehurd) [inline Wikipedia/Code/WMFImageController.swift]: Ohh that's just removing it from the mem cache... gotcha.

Comment (montehurd) [inline Wikipedia/Code/WMFImageController.swift]: Although... if we're not using the mem cache is there any point in using SDWebImage ;)

**Pull request merged**